### DISCUSSION TITLE
Port Forward (Shared) questions

### DISCUSSION THREAD

**Discussion by litsyneko** [coder/coder#14783] [General]
- This can be awkward because you're talking about it using a translator.

I opened the Minecraft port (25565) on the port share, but it works, but it doesn't seem to connect.

[Information]
> Ubuntu Server 24
> Nginx

**Comment by stirby** [comment]
Hey @whitehiyuri, are you trying to [share a port](https://coder.com/docs/networking/port-forwarding#sharing-ports) on your workspace in the UI publicly or forward it from your workspace to your machine? Also, what version of coder are you running?

**Comment by deansheather** [comment]
Sharing ports only allows access via the web, which requires the app to be serving HTTP or HTTPS traffic. We currently do not support raw TCP forwarding (which Minecraft requires) or UDP forwarding
